chef/cookbooks/provisioner/templates/default/autoyast.xml.erb

```diff
@@ -27,6 +27,13 @@
     </signature-handling>
     <storage/>
   </general>
+  <bootloader>
```

Comment:
we don't have that in crowbar-core/master so we shouldn't add it to the  SLE12 case on the backport then